dexbot/strategies/relative_orders.py

```diff
@@ -197,6 +201,9 @@ def calculate_order_prices(self):
                 # Calculate with quote amount if given
                 center_price = self.get_market_center_price(quote_amount=self.center_price_depth)
 
+            if self.cp_from_last_trade:
+                center_price = self.get_own_last_trade()['price']
```

Comment:
Add handling if `self.get_own_last_trade()` returned unusable result (like if it was no trades in account history).